project/AppDependencies.scala

```diff
@@ -6,26 +6,25 @@ import sbt._
 
 object AppDependencies {
 
-  val AkkaVersion      = "2.6.19"
-  val catsVersion      = "2.7.0"
-  val bootstrapVersion = "7.12.0"
+//  val AkkaVersion      = "2.6.19"
+  val catsVersion      = "2.9.0"
+  val bootstrapVersion = "8.4.0"
 
   val compile = Seq(
-    "uk.gov.hmrc"        %% "bootstrap-backend-play-28" % bootstrapVersion,
-    "com.lightbend.akka" %% "akka-stream-alpakka-xml"   % "3.0.4",
-    "com.typesafe.akka"  %% "akka-stream"               % AkkaVersion,
-    "org.typelevel"      %% "cats-core"                 % catsVersion,
+    "uk.gov.hmrc"      %% "bootstrap-backend-play-30" % bootstrapVersion,
+    "org.apache.pekko" %% "pekko-connectors-xml"      % "1.0.1",
+    "org.apache.pekko" %% "pekko-stream"              % "1.0.1",
+    "org.typelevel"    %% "cats-core"                 % catsVersion,
     // required for scalaxb
-    "org.scala-lang.modules" %% "scala-xml"                % "1.3.0",
     "org.scala-lang.modules" %% "scala-parser-combinators" % "1.1.2",
     "javax.xml.bind"          % "jaxb-api"                 % "2.3.1"
   )
 
   val test = Seq(
-    "uk.gov.hmrc"         %% "bootstrap-test-play-28" % bootstrapVersion,
-    "com.typesafe.play"   %% "play-test"              % current,
+    "uk.gov.hmrc"         %% "bootstrap-test-play-30" % bootstrapVersion,
+    "com.typesafe.play"   %% "play-test"              % "2.9.1",
     "com.vladsch.flexmark" % "flexmark-all"           % "0.36.8",
```

Comment:
```"com.vladsch.flexmark" % "flexmark-all"           % "0.36.8",```This is already with play-30. so we can remove it.

Reply:
if we found something that is already there with under test section https://github.com/hmrc/bootstrap-play/blob/0a1273f794ffcd1c993ac62fd70cea607d2c2b30/project/LibDependencies.scala#L67 . It is recommended to remove to avoid the dependency conflict.